tests/commonlib/kubernetes.py

```diff
@@ -304,13 +311,43 @@ def wait_for_resource(self, resource_type: str, name: str, status_list: list,
                               timeout_seconds=timeout,
                               **kwargs):
             if name in event["object"].metadata.name and event["type"] in status_list:
-                if (resource_type == RESOURCE_POD) and ('ADDED' in status_list) and (event['object'].status.phase == 'Pending'):
+                if (resource_type == RESOURCE_POD) and ('ADDED' in status_list) and (
+                        event['object'].status.phase == 'Pending'):
                     continue
                 w.stop()
                 return True
 
         return False
 
+    def pod_exec(self, name: str, namespace: str, command: list) -> str:
+        """
+        This function connects to pod and executes command
+        @param name: Pod name
+        @param namespace: Pod namespace
+        @param command: Command to be executed
+        @return: Executed command response
+        """
+
+        resp = stream(self.core_v1_client.connect_get_namespaced_pod_exec,
+                      name,
+                      namespace,
+                      command=command,
+                      stderr=True, stdin=False,
+                      stdout=True, tty=False,
+                      _preload_content=False)
```

Comment:
nit: usually when I split one arg by a new line, I split all of them

Reply:
fixed